analyticConfigurations/metric_meta-efs.json

```diff
@@ -0,0 +1,60 @@
+{
+  "analyticConfiguration": {
+    "metrics": [
+      {
+        "match": "aws.efs.burstcreditbalance.*",
+        "properties": {
+          "statistic": "MAX",
```

Comment:
I don't think these are counters. Take a look at https://docs.aws.amazon.com/efs/latest/ug/monitoring-cloudwatch.html for which statistics should be used. I believe the byte metrics should be SUM and the rest should be AVERAGE (the default). I also think all the types should be defaults.